clients.yml

```diff
@@ -1126,6 +1126,48 @@ clients:
   - 192.34.27.115
   - 2602:ffb1:0200:0:192:34:27:115
 
+- asn: "198585"
+  cfg:
+    filtering:
+      irrdb:
+        as_sets:
+        - AS198585:AS-MAIN
+  description: "WLTCS LLC"
+  ip:
+  - 192.34.27.117
+  - 2602:ffb1:0200:0:192:34:27:116
+
+- asn: "203913"
+  cfg:
+    filtering:
+      irrdb:
+        as_sets:
+        - AS203913:AS-ROHMAN
+  description: "Nurohman"
+  ip:
+  - 2602:ffb1:0200:0:192:34:27:117
+
+- asn: "199762"
+  cfg:
+    filtering:
+      irrdb:
+        as_sets:
+        - AS199762:AS-KOMOREBI
+  description: "Adita Tarkono Putra"
+  ip:
+  - 2602:ffb1:0200:0:192:34:27:118
+
+- asn: "150249"
+  cfg:
+    filtering:
+      irrdb:
+        as_sets:
+        - AS150249:AS-ATHARVA
+  description: "PT Atharva Telematika Persada"
+  ip:
+  - 192.34.27.116
```

Comment:
lets keep the IPv4 and IPv6 addresses in sync - so this should be 192.34.27.119/24

Reply:
Actually, please assign v4 addresses to each peer, even if they are not using v4, it breaks the website formatting otherwise.

Reply:
Neon corrected it, is fixed now.